requirements.txt

```diff
@@ -48,13 +48,14 @@ psycopg2-binary~=2.9.8
 pyasn1==0.1.9
 pycparser==2.14
 PyJWT==2.4.0
+pytz
```

Comment:
Should we pin a pytz version? There is a PR in 1.7.x that also adds pytz with 2014.1 pinned.

https://github.com/openlibhums/janeway/pull/4468/files